RTProtocol.cs

```diff
@@ -423,31 +452,35 @@ public int ReceiveRTPacket(out PacketType packetType, bool skipEvents = true, in
                     while (receivedTotal < frameSize)
                     {
                         // As long as we haven't received enough data, wait for more
-                        received = mNetwork.Receive(ref data, receivedTotal, frameSize - receivedTotal, false, -1);
-                        if (received <= 0)
+                        response = mNetwork.Receive(ref data, receivedTotal, frameSize - receivedTotal, false, 5000000);
```

Comment:
What unit is this in? ms? (just a bit confused about the values chosen)

Reply:
It is in microseconds. So the timeout is 5 sec. I have moved it to a constant.